bin/install-wp-tests.sh

```diff
@@ -55,7 +55,7 @@ get_db_connection_flags() {
 			EXTRA_FLAGS=" --host=$DB_HOSTNAME --protocol=tcp"
 		fi
 	fi
-	echo "--user=$DB_USER --password=$DB_PASS $EXTRA_FLAGS";
+	echo "--user=$DB_USER --password=$DB_PASS $EXTRA_FLAGS --skip_ssl";
```

Comment:
Without this I couldn't run the PHP tests locally, you can test if they run without this and if positive I assume this is in my local and remove it.